.github/workflows/sycl-nightly.yml

```diff
@@ -50,6 +50,14 @@ jobs:
             target_devices: ext_oneapi_hip:gpu
             tests_selector: e2e
 
+          - name: Intel Build Tests Only
+            runner: '["Linux", "build"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:alldeps
+            image_options: -u 1001 --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:cpu
+            tests_selector: e2e
+            extra_lit_opts: --param split-mode=build
+
           - name: Intel L0 GPU
             runner: '["Linux", "gen12"]'
```

Comment:
the other e2e runs aren't using the tests build from the previous step yet right? if it does i dont see how that's happening :)

Reply:
Yes, this pr does not include any of the running side of the split mode. Basically what we would have to do is make this job act similar to how the job for building the compiler works by spitting out an artifact which would include the built tests, and then on the running side we would take that artifact and run the tests with `--param split-mode=run`. That requires a bit more changes with the github ci, which im not too familiar so I wanted to save it for future prs.

Reply:
got it, probably udit/myself/andrei can help with that if you are stuck